cmd/podman/build.go

```diff
@@ -186,7 +186,9 @@ func buildCmd(c *cliconfig.BuildValues) error {
 	if err != nil {
 		return nil
 	}
-	contextDir := ""
+	// Ignore error if os.Getwd fails. contextDir.
+	// contextDir could be updated later.
+	contextDir, _ := os.Getwd()
```

Comment:
I would prefer putting it in the `else` case. In case we're not updating it, we would mask this error. It's unlikely to ever occur but it seems cleaner to not ignore it.

Reply:
Maybe...
```golang
if len(cliArgs) == 0 {
   a,b := os.Getwed()
} else {
   //big body below
}